.ci/manywheel/build_common.sh

```diff
@@ -136,6 +136,12 @@ else
     USE_KINETO=1
 fi
 
+# Fix for exposing statically linked libstdc++ CXX11 ABI symbols.
+# see: https://github.com/pytorch/pytorch/issues/133437
+LIBNONSHARED=$(gcc -print-file-name=libstdc++_nonshared.a)
+nm -g $LIBNONSHARED | grep " T " | grep recursive_directory_iterator | cut -c 20-  > weaken-symbols.txt
+objcopy --weaken-symbols weaken-symbols.txt $LIBNONSHARED $LIBNONSHARED
```

Comment:
It should be part of the docker container build, shouldn't it? (As you patch the library)

Also, what's your test plan?

Reply:
Will add test to check_binary.sh